main_server/local_server/bmlt.js

```diff
@@ -16,22 +16,26 @@ var militaryToStandard = function(value) {
             return value;
         }
         else {
-            if(value.length == 8){ //If value is the expected length for military time then process to standard time.
-                var hour = value.substring ( 0,2 ); //Extract hour
-                var minutes = value.substring ( 3,5 ); //Extract minutes
-                var identifier = 'AM'; //Initialize AM PM identifier
+            if (value.length == 8) { //If value is the expected length for military time then process to standard time.
+                value = value.split(':'); // convert to array
+                // fetch
+                var hours = Number(value[0]);
+                var minutes = Number(value[1]);
```

Comment:
Since you are not manipulating minutes, why not just leave it a string, instead of re-converting to a string on line 35?